clang/test/Driver/fp-accuracy.c

```diff
@@ -38,7 +38,11 @@
 
 // RUN: not %clang -fno-math-errno -ffp-accuracy=low:[sin,cos] \
 // RUN: -ffp-accuracy=high %s 2>&1  \
-// RUN: | FileCheck %s --check-prefix=WARN
+// RUN: | FileCheck %s --check-prefix=WARN1
+
+// RUN: not %clang -fno-math-errno -ffp-accuracy=low:[sin,cos] \
+// RUN: -ffp-accuracy=high:[cos,tan] %s 2>&1  \
+// RUN: | FileCheck %s --check-prefix=WARN2
```

Comment:
Can you add a case for %clang -fno-math-errno -ffp-accuracy=low -ffp-accuracy=high:cos?

Also %clang -fno-math-errno -ffp-accuracy=high -ffp-accuracy=default:cos?